src/commands/MooCommand.ts

```diff
@@ -0,0 +1,37 @@
+import { Message } from 'discord.js';
+import PrefixCommand from './PrefixCommand';
+import { SingleMention } from '../mentions/SingleMention';
+
+export default class PingCommand extends PrefixCommand {
+	public readonly aliases = ['moo', 'test'];

```

Comment:
The "test" alias is already taken by !jira ping

Reply:
switched it to "cow"